src/art/avatar/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Belgium Flag</title>
 </head>
```

Comment:
Please explain why you change another committer's code.

Reply:
I fixed the issues. Kindly reopen this PR so that I can commit